upgrading/change-log.md

```diff
@@ -1,35 +1,51 @@
 ---
 title: Changelog | FusionCharts
-description: This section talks about the change in features and attributes with latest released version.
-heading: Version 3.20.x
+description: This section discusses the change in features and attributes with the latest released version.
+heading: Version 3.21.x
 ---
 
-<h2 class="sub-heading">Version 3.20</h2>
+<h2 class="sub-heading">Version 3.21</h2>
+
+<p class="release-date">28th September, 2023</p>
 
-<p class="release-date">24th March, 2023</p>
 
 <h4>New Features</h4>
 
-- FusionCharts version 3.20 introduces a new method, `_changeXAxisCoordinates,` which allows users to change the x-axis to be centered-aligned upon legend or data interaction automatically. 
-- FusionCharts version 3.20 updated `Angular` integration to support Angular versions 14 and 15.
-- FusionCharts version 3.20 updated `React Native` plugin to support React Native versions 0.68 and 0.69.
-- FusionCharts version 3.20 updated the wrappers for `Python`, `C#`, and `Java SDK` to the latest supported versions. 
-- FusionCharts version 3.20 added the new property `orientedTextAlign`, which allows users to align the axis text.
+- FusionCharts version 3.21 updated the `Ramda` dependency package version to the latest version, version 0.29. 
+- FusionCharts version 3.21 introduces the new function that removes any cross-site scripting that might be causing errors in your code.
+- FusionCharts version 3.21 introduces new maps for all 24 Argentinian departments, all 26 Peruvian departments, and all 7 provinces from Costa Rica. 
```

Comment:
Previously Costa Rican departments were used in another file. Just highlighting it in case it needs consistency.